src/meds_torch/models/eic_forecasting.py

```diff
@@ -117,9 +179,10 @@ def __init__(self, vocab_size: int, dist_sync_on_step=False):
         self.vocab_size = vocab_size
 
         self.auroc = MulticlassAUROC(num_classes=vocab_size, average="weighted", thresholds=100)
-        self.top_1_accuracy = MulticlassAccuracy(num_classes=vocab_size, top_k=1)
-        self.top_5_accuracy = MulticlassAccuracy(num_classes=vocab_size, top_k=5)
-        self.top_10_accuracy = MulticlassAccuracy(num_classes=vocab_size, top_k=10)
+        self.top_k_acc = top_k_acc
+        self.accuracy_metrics = MetricCollection(
```

Comment:
Why not put AUROC into the metric collection too? Then just have all metrics in one collection?